dist/projects.html

```diff
@@ -29,23 +29,23 @@
             <div class="row mb-5">
                 <div class="col-md-8 col-xl-6 text-center mx-auto">
                     <h2 class="fw-bold">Projects</h2>
-                    <p class="text-muted">Learn about the newest projects that Subsurface Studios has released</p>
+                    <p class="text-muted">Here are our latest and greatest projects! Check back later to see even more, or click on one of the projects to find out more!</p>
```

Comment:
I updated this to be a bit more friendly. Let me know if you want this rolled back and I will.